llpc/context/llpcCompiler.cpp

```diff
@@ -1504,9 +1517,14 @@ Result Compiler::BuildGraphicsPipeline(const GraphicsPipelineBuildInfo *pipeline
   Result cacheResult = Result::ErrorUnknown;
 
   if (!buildingRelocatableElf) {
-    if (m_cache)
+    if (m_cache) {
       cacheResult = lookUpCaches(userCache, &hashId, &elfBin, &cacheEntry);
-    else
+      if (cacheResult == Result::Success) {
+        pipelineOut->pipelineCacheAccess = CacheAccessInfo::CacheHit;
+      } else {
+        pipelineOut->pipelineCacheAccess = CacheAccessInfo::CacheMiss;
+      }
```

Comment:
I think this might be in the wrong place.  The `lookUpShaderCaches` is also a cache lookup, and should set the `pipelineCacheAccess`.  Can you move setting this value to include the lookup in the other cache?